codegens/curl/test/unit/convert.test.js

```diff
@@ -34,41 +34,6 @@ describe('curl convert function', function () {
       });
     });
 
```

Comment:
@webholik Does e2e test assert that whatever cURL that got generated from the request has certain body? (i.e. raw body is escaped correctly).

Reply:
@VShingala E2E tests ensure that the curl code we generated behaves exactly as Postman App. So it will ensure that the body is escaped properly (otherwise the response from `postman-echo` will be different)